tests/cpp/utils.cpp

```diff
@@ -816,4 +816,12 @@ bool isVectorized(TensorView* tv) {
   return false;
 }
 
+std::string macroToString(const MmaMacro macro) {
+  std::stringstream ss;
+  ss << "m" << getM(macro);
+  ss << "_n" << getN(macro);
+  ss << "_k" << getK(macro);
+  return ss.str();
+}
```

Comment:
We actually already have `toString(MmaMacro)` which is exactly like this but will prepend `Hopper_`. Could use that instead or keep this since it's not very big.

Reply:
I'll change it later.